jax/core.py

```diff
@@ -1748,7 +1748,9 @@ def subst_axis_names_jaxpr(jaxpr: Union[Jaxpr, ClosedJaxpr], subst: AxisSubst):
   if isinstance(jaxpr, ClosedJaxpr):
     consts = jaxpr.consts
     jaxpr = jaxpr.jaxpr
-  var_map: Dict[Var, Var] = {}
+  var_map: Dict[Var, Var] = {
+      unitvar: unitvar
```

Comment:
Why is this necessary? unitvar is specially handled in `subst_axis_names_var`

Reply:
Done.